app/src/aws-exports.js

```diff
@@ -0,0 +1,19 @@
+// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.
+
+const awsmobile = {
+    "aws_project_region": "us-west-2",
```

Comment:
This shouldn't get checked in anymore since it should be generated through amplify. We'll need to ensure it can be generated and used correctly by the amplify lib